tests/integration/pure_cairo_vm/cheatcodes/expect_events/expect_event_cairo_cheatcode_test.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+
+import pytest
+
+from protostar.testing.test_results import TestCaseResult
+from protostar.testing.testing_summary import TestingSummary
+from tests.integration.conftest import (
+    CreateProtostarProjectFixture,
+    assert_cairo_test_cases,
+)
+from tests.integration._conftest import ProtostarFixture
+from tests.integration.pure_cairo_vm.conftest import CONTRACTS_PATH
+
+TEST_PATH = Path(__file__).parent / "expect_events_test.cairo"
```

Comment:
That variable is redundant

Reply:
[remove constant](https://github.com/software-mansion/protostar/pull/1412/commits/2222f1c7ac8d829fc9f8a4a2f57d742a9e91a57d)